webots_ros2_control/CHANGELOG.rst

```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package webots_ros2_control
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.0.6 (2021-05-21)
```

Comment:
Just a question, is the version `1.0.6` correct here?
If yes, shouldn't we add a line for version `1.1.0`?

Reply:
You are right, I opened a PR for it